python/katana/bug/environment.py

```diff
@@ -203,14 +205,16 @@ def is_interactive():
     return hasattr(sys, "ps1")
 
 
-def capture_environment(filename: Optional[Union[str, Path, Any]] = None):
+def capture_environment(filename: Optional[Union[str, Path, Any]] = None, **kwargs):
+    # pylint: disable=unused-argument
     """
     Capture the execution and build environment in as much detail as reasonably possible
     and store it to a file. This is used for bug reporting.
 
     :param filename: The file name for the captured environment information.
         (Default: an auto generated file in the system temporary directory)
     :type filename: str or Path or a file-like object or None
+    :key client: Required if using remote enviroment. Expects katana.remote.Client
```

Comment:
I worry about documenting this here because this documentation is related to code that isn't here. But I also see the value in it. Hmm. What do you think?

Reply:
I would say keep it, the user can just ignore this since it specify when required. I think the remote users will recognize since they will do imports with remote and also build a remote client so it should click with them more than other users.

Reply:
Maybe we say required when using enterprise remote client. Else maybe we have a separate capture_environment for enterprise with it's own docstring that acts as a pass through to the open source one but the enterprise will have the correct docstring. I don't know how I like hiding it from the user, feel like it should be documented somewhere atleast.

Reply:
I think this is fine.

BTW. You should put full sentences in doc strings. What you wrote here is very stilted because of the lack of articles and stuff.

Reply:
done